main.go

```diff
@@ -25,13 +26,16 @@ var (
 )
 
 func main() {
+	cmd.Execute()
+
```

Comment:
This looks like you're forcing the tool to be command line?

Reply:
This was in the documentation, we have to add this cmd.execute() into our main method in order to make the commands work.